recipes/_setup_python.rb

```diff
@@ -53,4 +64,7 @@
     # FIXME: https://github.com/poise/poise-python/issues/133
     pip_version '18.0'
   end
+  python_virtualenv node['cfncluster']['virtualenv'] do
+      pip_version '18.0'
+  end
```

Comment:
Can we clean up all this logic and just create a python virtual env valid for all the OSs?

Reply:
done